source/lac/lapack_full_matrix.cc

```diff
@@ -1932,6 +1933,7 @@ LAPACKFullMatrix<number>::compute_eigenvalues(const bool right, const bool left)
 
   // resize workspace array
   work.resize(lwork);
+  real_work.resize(lwork);
```

Comment:
This was missing, combined with the change I had to make in the function calling the eigenvalue function - so it seems `compute_eigenvalues()` never worked with complex-valued matrices.